### DISCUSSION TITLE
Rate limit plugin with Distributed cache policy

### DISCUSSION THREAD

**Discussion by SSukhada** [Kong/kong#11724] [Help]
Hi ,

The rate limit plugin works with the Redis database if redis policy is enabled. Does rate limit work with distributed cache (for example, Infinispan) as well when policy is selected as redis?

**Comment by StarlightIbuki** [comment]
The rate-limiting plugin interfaces with Redis and it talks Redis restful API when configured with the Redis strategy, so direct access to Infinispan is not supported.
